src/main/java/org/omnifaces/util/Components.java

```diff
@@ -248,6 +248,18 @@ public static <T> T getAttribute(UIComponent component, String name) {
         return (T) component.getAttributes().get(name);
     }
 
+    /**
+     * Set the attribute value on the specified component for the specified name.
+     * @param <T> The expected value type.
+     * @param component The component on which to set the attribute.
+     * @param name The name of the attribute.
+     * @param value The value of the attribute.
+     * @since 4.5
+     */
+    public static <T> void setAttribute( UIComponent component , String name , T value ) {
+        component.getAttributes().put(name,value);
```

Comment:
Can you please follow the same code format as seen in existing source code? The whitespace is off here.